mirdata/datasets/dagstuhl_choirset.py

```diff
@@ -0,0 +1,465 @@
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_stm_path)
+
+    @property
+    def audio_str(self) -> Tuple[np.ndarray, float]:
+        """The audio for the room mic (right channel)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_str_path)
+
+    @property
+    def audio_stl(self) -> Tuple[np.ndarray, float]:
+        """The audio for the room mic (left channel)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_stl_path)
+
+    @property
+    def audio_rev(self) -> Tuple[np.ndarray, float]:
+        """The audio for the room mic with artifical reverb (mono mixdown)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_rev_path)
+
+    @property
+    def audio_spl(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The audio for the piano accompaniment DI (left channel)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_spl_path)
+
+    @property
+    def audio_spr(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The audio for the piano accompaniment DI (right channel)
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_spr_path)
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+
+        beat_data = [(self.beat, "beat")] if self.beat else None
+
+        return jams_utils.jams_converter(
+            audio_path=self.audio_stm_path, beat_data=beat_data
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load a Dagstuhl ChoirSet audio file.
+
+    Args:
+        audio_path (str): path pointing to an audio file
+
+    Returns:
+        * np.ndarray - the audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=22050, mono=True)
+
+
+@io.coerce_to_string_io
+def load_f0(fhandle: TextIO) -> annotations.F0Data:
+    """Load a Dagstuhl ChoirSet F0-trajectory.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to F0 file
+
+    Returns:
+        F0Data Object - the F0-trajectory
+    """
+    times = []
+    freqs = []
+    confs = []
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        times.append(float(line[0]))
+        freqs.append(float(line[1]))
+        if len(line) == 3:
+            confs.append(float(line[2]))
+        else:
+            confs.append(float(1.0))
+
+    times = np.array(times)
+    freqs = np.array(freqs)
+    confs = np.array(confs)
+    return annotations.F0Data(times, freqs, confs)
```

Comment:
to fix the mypy error, replace these 4 lines with just:

`return annotations.F0Data(np.array(times), np.array(freqs), np.array(confs))`